projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts

```diff
@@ -149,7 +149,8 @@ export class FancyTreeComponent implements OnInit, AfterViewInit, OnDestroy {
         primaryCategory: _.get(this.editorService, 'editorConfig.config.primaryCategory'),
         objectType: _.get(this.editorService, 'editorConfig.config.objectType'),
         metadata: {
-          name: child.name
+          name: child.name,
+          mimeType: "application/vnd.sunbird.questionset"
```

Comment:
and this mimeType should be taken from the category definition config based on the level.